### PR TITLE
Log_Injection: improve formatting

### PR DESCRIPTION
**Pull request by alxndrsn**

### CODE REVIEW

File: pages/attacks/Log_Injection.md [hunk 1]
```diff
@@ -54,32 +54,32 @@ from a request object. If the value fails to parse as an integer, then
...
-    ...
-    String val = request.getParameter("val");
-    try {
-        int value = Integer.parseInt(val);
-    }
-    catch (NumberFormatException) {
-        log.info("Failed to parse val = " + val);
-    }
-    ...
+...
```
Comment (kingthorin) [inline pages/attacks/Log_Injection.md:57]: add java rendering hint?

Comment (alxndrsn) [reply pages/attacks/Log_Injection.md:57]: :+1:

Is there any preference for bundling all changes for a file together vs. separating different types of changes?

Comment (alxndrsn) [reply pages/attacks/Log_Injection.md:57]: Added :+1:

Comment (alxndrsn) [reply pages/attacks/Log_Injection.md:57]: Added :+1:

**Review by kingthorin — APPROVED** [review]

**Pull request merged**